Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 ruby "2.7.1"
 
+gem "bcrypt", "3.1.13"
+
 gem "config"
 
 gem "rails-i18n"
```

Comment:
các gem như này viết liền vào e, đừng cách 1 dòng trawsng
gem bcrypt để làm gì  e?

Reply:
Đây là gem để dùng thuật toán Bcrypt mã hoá cho mật khẩu của User ạ